daemon/mgr/cri_network.go

```diff
@@ -0,0 +1,115 @@
+package mgr
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cri-o/ocicni/pkg/ocicni"
+	"github.com/sirupsen/logrus"
+
+	"github.com/alibaba/pouch/daemon/config"
+
+	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+)
+
+// CniMgr as an interface defines all operations against CNI.
+type CniMgr interface {
+	// Name returns the plugin's name. This will be used when searching
+	// for a plugin by name, e.g.
+	Name() string
+
+	// SetUpPod is the method called after the sandbox container of the
+	// pod has been created but before the other containers of the pod
+	// are launched.
+	SetUpPod(config *runtime.PodSandboxConfig, id string, netnsPath string) error
+
+	// TearDownPod is the method called before a pod's sandbox container will be deleted.
+	TearDownPod(config *runtime.PodSandboxConfig) error
```

Comment:
I think `TearDownPodNetwork` is better? 

CNI-Manager is responsible for the network management of pod, right?